pages/_app.js

```diff
@@ -1,14 +1,15 @@
-import GlobalStyle from "@/styles";
+import GlobalStyle from "../styles";
+
 import Head from "next/head";
 
-export default function App({ Component, pageProps }) {
+export default function App({ Component, pageProps: { pageProps } }) {
   return (
     <>
       <GlobalStyle />
       <Head>
```

Comment:
Just a suggestion:

```suggestion
      <Head>
      <title>GraffStar</title>
      </Head>
      <h1>GraffStar</h1>
      <Component {...pageProps} />
      <Navbar />
```

If you put your heading and navbar into the App component, they get rendered on every page and you don't need to duplicate them.